src/dashboard/start.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# todo : mode debug or prod
+#python manage.py flush --no-input
+#python manage.py collectstatic --noinput
+python manage.py migrate
+# todo : add the port (8030) in .env
+python3 manage.py runserver 0.0.0.0:8030
```

Comment:
You can leave the default port (`8000`) and update the mapping in the docker-compose file (see corresponding comment):

```suggestion
python3 manage.py runserver 0.0.0.0
```

Reply:
And keep in mind that for production, the port value should be determined by the `${PORT}` environment variable.

Reply:
> You can leave the default port (`8000`) and update the mapping in the docker-compose file (see corresponding comment):

with django runserver, you can't specify the address without the port...